test/unit/widgets/peoplePicker.test.ts

```diff
@@ -258,10 +348,12 @@ describe('GroupBuilder.render', () => {
   it('exists', () => {
     expect(new GroupBuilder().render).toBeInstanceOf(Function)
   })
+
   it('runs', () => {
+    jest.clearAllMocks()
```

Comment:
In general, might want to move this into an `afterEach` block, to make sure it runs after all tests. In this case I don't think it's necessary though, but might want to do it when we add more tests that need to clear all mocks.

Reply:
I'd set up Jest to do that everywhere - not sure why it's not on by default (since you want unit tests to be independent).

https://jestjs.io/docs/en/configuration#clearmocks-boolean